praw/models/reddit/live.py

```diff
@@ -281,6 +281,55 @@ def close(self):
         url = API_PATH['live_close'].format(id=self.thread.id)
         self.thread._reddit.post(url)
 
+    def update(self, **settings):
```

Comment:
I read other your comments again. Now I think I should write

```
def set(self, title, description='', nsfw=False, resources='',
           **other_settings)
```

instead of `update()`.